crates/bevy_pbr/src/ssao/mod.rs

```diff
@@ -127,8 +127,9 @@ impl Plugin for ScreenSpaceAmbientOcclusionPlugin {
             .add_render_graph_edges(
                 CORE_3D,
                 &[
-                    // PREPASS -> SCREEN_SPACE_AMBIENT_OCCLUSION -> MAIN_PASS
+                    // PREPASS -> DEFERRED -> SCREEN_SPACE_AMBIENT_OCCLUSION -> MAIN_PASS
                     bevy_core_pipeline::core_3d::graph::node::PREPASS,
+                    bevy_core_pipeline::core_3d::graph::node::DEFERRED_PREPASS,
```

Comment:
They are primarily ordered here: https://github.com/DGriffin91/bevy/blob/0968b9fe6866c9b5b2ca5449e4fd2a1a212ed787/crates/bevy_core_pipeline/src/core_3d/mod.rs#L143-L144

This just makes sure SSAO if after both in the event a user has a custom render graph. There could alternatively be a `END_PREPASS` similarly to `END_MAIN_PASS` or something along those lines.

Reply:
why does ssao need to run after the deferred pass?

Reply:
So it has all of the opaque depth and normals. If it were to run between the prepass and deferred it would only have the depth and normals from forward+prepass materials.

Reply:
makes sense.

this is probably a bit anal but could you separate it so you have deferred->ssao and prepass->ssao? ssao should need to know about the ordering of the prepass vs deferred prepass

Reply:
I feel like in that case something equivalent to `END_PREPASS` that works like the `END_MAIN_PASS ` would be cleaner. (though I don't love the name `END_PREPASS`)

Reply:
I think this may be the only outstanding issue I see in the comments...

Reply:
I decided to just add a sync point. I think this will be particularity needed in the future since we'd have passes running after deferred that would still want to be before anything that wants to *use* the prepasses or deferred. (And the `CopyDeferredLightingIdNode` would already need this for some things anyway)